nullaway/src/main/java/com/uber/nullaway/AbstractConfig.java

```diff
@@ -34,6 +34,7 @@
 import javax.annotation.Nullable;
 
 /** abstract base class for null checker {@link Config} implementations */
+@SuppressWarnings("NullAway") // TEMPORARY
```

Comment:
Before landing, I want to just get rid of this class and push its functionality down to `ErrorProneCLIFlagsConfig` (in a separate PR)